sqlFile.js

```diff
@@ -0,0 +1,43 @@
+var sqlite3 = require("sqlite3").verbose();
+var db = new sqlite3.Database("url-shortner.db");
+
+const DBSOURCE = "db.sqlite3";
+
+function createUrlTable() {
+  db.run("CREATE TABLE IF NOT EXISTS urlTable (fullUrl TEXT, shortCode TEXT)");
```

Comment:
Since we know we are going to search for rows by the `shortCode` field, you would typically optimize the database by creating an index for the table so that lookups for shortCodes are fast.

Reply:
You should probably also make the shortCode field a `unique` field, with a database constraint, so that there is no possible way to accidentally store a duplicate short code. It's best if your database structure is designed to be as strict as possible so that you never introduce invalid data, like duplicate short codes.